app/src/main/resources/application.yml

```diff
@@ -121,18 +150,6 @@ lhAPIProvider:
   apiKey: "${LH_VRO_API_KEY:placeholderKey}"
   validateToken: "${VALIDATE_TOKEN:NO}"
 
-
-
-slack:
-  exception:
-    webhook: "${SLACK_EXCEPTION_WEBHOOK:#{null}}"
-    channel: "${SLACK_EXCEPTION_CHANNEL:#{null}}"
-
-opentracing:
-  zipkin:
-    enabled: false
-    http-sender.baseUrl: http://zipkin.istio-system.svc.cluster.local:9411
-
```

Comment:
The values in this block is dependent on the env, so it's been moved to env-specific `application-*.yml` files.